src/task/MonitorTask.ts

```diff
@@ -436,12 +436,9 @@ export default class MonitorTask
     if (attendeeId === presentAttendeeId && present && !this.presenceHandlerCalled) {
       this.presenceHandlerCalled = true;
       this.context.attendeePresenceDurationMs = Date.now() - this.context.startAudioVideoTimestamp;
-      /* istanbul ignore else */
-      if (this.context.eventController) {
-        this.context.eventController.publishEvent('attendeePresenceReceived', {
-          attendeePresenceDurationMs: this.context.attendeePresenceDurationMs,
-        });
-      }
+      this.context.eventController?.publishEvent('attendeePresenceReceived', {
```

Comment:
This is not available before Node 14. Could we update the README to replace the Node 12 or higher.

Reply:
Still works for Node 12 when you install through NPM, as Kyu mentioned in Slack